enterprise/server/remote_execution/action_merger/action_merger.go

```diff
@@ -117,14 +130,69 @@ func RecordClaimedExecution(ctx context.Context, rdb redis.UniversalClient, exec
 }
 
 // Records a hedged execution in Redis.
-func RecordHedgedExecution(ctx context.Context, rdb redis.UniversalClient, adResource *digest.ResourceName) error {
+func RecordHedgedExecution(ctx context.Context, rdb redis.UniversalClient, adResource *digest.ResourceName, groupIdForMetrics string) error {
 	key, err := redisKeyForPendingExecutionID(ctx, adResource)
 	if err != nil {
 		return err
 	}
+
 	return rdb.HIncrBy(ctx, key, executionCountKey, 1).Err()
 }
 
+// Records a merged execution.
+func RecordMergedExecution(ctx context.Context, rdb redis.UniversalClient, adResource *digest.ResourceName, groupIdForMetrics string) error {
```

Comment:
nit: mention where the execution is being recorded?

Reply:
done